### PR TITLE
feat(replays): Update replay docs with web vital info

### PR DESCRIPTION
**Pull request by c298lee**
Relates to:
* https://github.com/getsentry/sentry/issues/69881

### CODE REVIEW

File: docs/product/explore/session-replay/web/replay-details.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ Every replay has a detailed view that contains the embedded video player and ric
...
+* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [web vitals](/product/insights/web-vitals/). User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. Web vitals are augmented with the selector to help users identify which elements are contributing to their web vital scores. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```
Comment (michellewzhang) [inline docs/product/explore/session-replay/web/replay-details.mdx:16]:
```suggestion
* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [web vitals](/product/insights/web-vitals/). User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```

**Review by michellewzhang — APPROVED** [review]

File: docs/product/explore/session-replay/web/replay-details.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ Every replay has a detailed view that contains the embedded video player and ric
...
+* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [web vitals](/product/insights/web-vitals/). User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```
Comment (jas-kas) [inline docs/product/explore/session-replay/web/replay-details.mdx:16]: > Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores

Should we move this sentence to be immediately following the web vitals link? That would seem to make sense

**Review by jas-kas — APPROVED** [review]

File: docs/product/explore/session-replay/web/replay-details.mdx [hunk 3]
```diff
@@ -13,7 +13,7 @@ Every replay has a detailed view that contains the embedded video player and ric
...
+* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [web vitals](/product/insights/web-vitals/). Web vitals are augmented with the selector to help users identify which elements are affecting their web vital scores. User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```
Comment (jas-kas) [inline docs/product/explore/session-replay/web/replay-details.mdx:16]:
```suggestion
* **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user clicks (including rage and dead clicks), user inputs, [page navigations, page loads](/product/performance/transaction-summary/#what-is-a-transaction), as well as [custom breadcrumbs](/product/sentry-basics/integrate-backend/configuration-options/#breadcrumbs) set by the organization and [Web Vitals](/product/insights/web-vitals/). Web Vitals are augmented with the selector to help users identify which elements are affecting their Web Vital scores. User clicks (including rage and dead clicks), are augmented with surrounding HTML source code to help users understand which event was targeted. These breadcrumbs are synced with the replay player and will auto-scroll as the video plays.
```

**Pull request merged**